usr/bin/xfce4-desktop-service

```diff
@@ -74,6 +75,7 @@ class signal_handlers(dbus.service.Object):
     def Exit(self):
         """Quit"""
         mainloop.quit()
+        return None
```

Comment:
You don't need to explicitly `return None`. You can just leave the entire `return` call off and it should work fine.